app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue

```diff
@@ -262,6 +262,9 @@ export default {
         this.fetchPortalAndItsCategories();
       }
     },
+    '$route.params.locale'() {
```

Comment:
Can we do this with a bus event instead of watching a deep object? WDYT? I do not have strong opinions on either approach here, but a bus event seems more explicit